src/Trakx.Shrimpy.ApiClient.Tests/Trakx.Shrimpy.ApiClient.Tests.csproj

```diff
@@ -12,16 +12,16 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="DotNetEnv" Version="2.3.0" />
-    <PackageReference Include="FluentAssertions" Version="6.8.0" />
+    <PackageReference Include="DotNetEnv" Version="2.5.0" />
```

Comment:
Can remove these dependencies, since they are already installed in `common` packages:
- DotNetEnv
- FluentAssertions
- Serilog.Sinks.XUnit